### ISSUE TITLE
_C_ops.gaussian_random is freezing

### ISSUE THREAD

**Issue by tempbrucefu** [PaddlePaddle/PaddleDetection#6007]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleDetection/issues)，没有报过同样bug。I have searched the [issues](https://github.com/PaddlePaddle/PaddleDetection/issues) and found no similar bug report.

### bug描述 Describe the Bug

new nn.Conv2D(
            in_channels=ch_in,
            out_channels=ch_out,
            kernel_size=filter_size,
            stride=stride,
            padding=padding,
            groups=groups,
            bias_attr=False)
going forever, it is freezing at 
 out_var = _C_ops.gaussian_random(
                'shape', var.shape, 'dtype', out_dtype, 'mean', self._mean,
                'std', self._std_dev, 'seed', self._seed, 'use_mkldnn', False)

### 复现环境 Environment

GPU Compute Capability: 8.6, Driver API Version: 11.6, Runtime API Version: 10.2
cuDNN Version: 7.6.

paddledet               2.4.0
paddleocr               2.5
paddlepaddle            2.2.2
paddlepaddle-gpu        2.3.0

### 是否愿意提交PR Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by yghstill** [comment]
@tempbrucefu 请问是跑哪个模型，具体的报错可以贴的更加完整些吗？

**Comment by tempbrucefu** [comment]
just tried:

from paddle.nn import Conv2D
test = Conv2D(in_channels=3, out_channels=16,
        kernel_size=3,
        stride=0,
        padding=1,
        groups=1,
        bias_attr=False)
#this will never be ended.

**Comment by zzl221000** [comment]
一样的问题
按照paddleocr教程训练卡住

`python tools/train.py -c configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml -o Global.pretrained_model=./pretrain_models/ch_PP-OCRv3_rec_train/best_accuracy`

paddle相关库
```
paddlebfloat 0.1.2
paddleocr 2.5.0.2
paddlepaddle-gpu 2.3.0
```

cuda
```
W0524 18:44:56.719388  7024 gpu_context.cc:278] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.3, Runtime API Version: 10.2
W0524 18:44:56.719388  7024 gpu_context.cc:306] device: 0, cuDNN Version: 7.6.
```

**Comment by zzl221000** [comment]
> 一样的问题 按照paddleocr教程训练卡住
> 
> `python tools/train.py -c configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml -o Global.pretrained_model=./pretrain_models/ch_PP-OCRv3_rec_train/best_accuracy`
> 
> paddle相关库
> 
> ```
> paddlebfloat 0.1.2
> paddleocr 2.5.0.2
> paddlepaddle-gpu 2.3.0
> ```
> 
> cuda
> 
> ```
> W0524 18:44:56.719388  7024 gpu_context.cc:278] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.3, Runtime API Version: 10.2
> W0524 18:44:56.719388  7024 gpu_context.cc:306] device: 0, cuDNN Version: 7.6.
> ```
github文档过时了，需要按照官网文档进行安装
